### PR TITLE
Added Melon symbol + contractmap.json info + removed other listing

### PR DESCRIPTION
**Pull request by thehighfiveghost**

### CODE REVIEW

**Comment by flyswatter** [comment]
Sorry for the slowness, trying to keep the repo as clean as possible.

In the diff, it looks like we already had Melon in the directory, so there are two major changes here:

- Renaming `Melon Token` to `Melon` (sounds fine)
- Replacing the SVG with a PNG. Is this necessary? SVGs look much better on a wider variety of screens. Was our old logo bad?

**Comment by flyswatter** [comment]
Oh also, if the old image is bad, we should remove it from this repo, too.

File: contract-map.json [hunk 1]
```diff
@@ -1,4 +1,11 @@
 {
+  "0xBEB9eF514a379B997e0798FDcC901Ee474B6D9A1": {
+    "name": "Melon",
+    "logo": "MLNSymbol.png"
```
Comment (kumavis) [inline contract-map.json]: missing a comma

**Comment by kumavis** [comment]
@thehighfiveghost couple things to address, look forward to merging

**Pull request merged**